src/Pos/Launcher/Runner.hs

```diff
@@ -220,6 +223,7 @@ runServer mkTransport mkReceiveDelay mkL (OutSpecs wouts) withNode afterNode oq
             mkListeners mkL' ourVerInfo theirVerInfo
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo: "%build) ourVerInfo
+    notifyReady
```

Comment:
I hear you folks, and that was exactly my concern as well. Finding the right spot where to put this is a bit tricky, so I'm happy to take recommendations. Unfortunately I'm going on annual leave this afternoon before having the chance to coordinate with @avieth properly 😞 

What I will try to do then is to take my best shot hoping I will do the right thing 😉  (by which I mean that is not clear to me, by reading Alex's comments, wether he changed his mind after looking at #1318 or indeed we might move `notifyReady` after `allocateNodeResources` 😀 )